src/components/Api.js

```diff
@@ -0,0 +1,93 @@
+export class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._authToken = headers.authorization;
+    this._headers = headers;
+  }
+
+  /*User routes*/
+  //Get the current user's info
+  getUserInfo() {
+    return fetch(`${this._baseUrl}/users/me`, { headers: this._headers })
+      .then((res) => (res.ok ? res.json() : Promise.reject(`Error: ${res.status}`)))
+      .catch(console.error);
+  }
+
+  //Update your profile information
+  updateProfile(user) {
+    return fetch(`${this._baseUrl}/users/me`, {
+      method: "PATCH",
+      headers: this._headers,
+      body: JSON.stringify(user),
+    })
+      .then((res) => res.json())
```

Comment:
` .then((res) => res.json())`


See line 13 and in the task https://disk.yandex.ru/i/-GJvafZk2FTPHw